.storybook/main.ts

```diff
@@ -1,4 +1,5 @@
 import type { StorybookConfig } from '@storybook/nextjs';
+import path from 'path';
```

Comment:
```suggestion
import { resolve } from 'node:path';
```

Reply:
sorry I am new to this, should i sign off this commit in order to accept the change ?

Reply:
Yeah, that’s fine 🙃